Applications/Express.md

```diff
@@ -0,0 +1,76 @@
+## Express
+
+The express project is applying to become an incubating
+[Top Level Project of the Node.js Foundation][].
+
+### About Express, User base, Community and Ecosystem
+
+Express is one of the most popular web server frameworks available for Node.js.
+For many Node.js developers, express is their primary interface with Node.js
+on the server.
+
+Express has consistently been among the top packages installed via npm, with
+nearly 4.5 million downloads in just the past month.
+
+It is important to recognize that Express is not the only web framework in the
+Node.js ecosystem and that by incubating Express, neither the Node.js
+Foundation or Node.js core would be "picking a winner" or endorsing Express
+over any other overlapping or similar project.
+
+### Formation of an Express WG
+
+The express project consists of a single top level core repository
+(strongloop/express) that has dependencies on a large number of smaller
+independent components that are currently spread out across multiple
+independent GitHub repositories. This proposal moves only the
+strongloop/express GitHub repository under the Node.js GitHub organization
+(nodejs/express).
+
+If the proposal for incubating express as a top level project is accepted, a
+call for participation would be put out to existing Node.js GitHub organization
+members and an "Express WG" (nodejs/express team in the github organization)
+with commit rights to the nodejs/express repository would be created. This WG
+would also initially include existing contributors to the strongloop/express
+github repository who are not currently members of the Node.js GitHub
+organization.
+
+A secondary nodejs/express-release team would be created consisting of the
+subset of collaborators directly involved in the release process. These would
+be identified out of the members of the main Express WG (following the
+precedence of the nodejs/build team).
+
+While under incubation, there would be no formal "Express TC". If and when a
```

Comment:
If every member of the WG has a vote in contentious issues they are essentially the project TC? I guess I'm trying to figure out what rights a TC has that a WG does not? We tend to treat them more or less the same at the top level.